### PR TITLE
HuaweiAds: accept Alpha2 country code

### PR DESCRIPTION
**Pull request by clho40**
modified the function convertCountryCode to accept Alpha2 country code

### CODE REVIEW

**Review by onkarvhanumante — APPROVED** [review]

File: adapters/huaweiads/huaweiads.go [hunk 1]
```diff
@@ -771,7 +771,7 @@ func convertCountryCode(country string) (out string) {
 		return mappedCountry
 	}
 
-	if len(country) >= 3 {
+	if len(country) >= 2 {
```
Comment (gargcreation1992) [inline adapters/huaweiads/huaweiads.go:774]: could you please add the test case for this?

Comment (onkarvhanumante) [reply adapters/huaweiads/huaweiads.go:774]: @clho40  could you address this comment to move forward with this PR

Comment (clho40) [reply adapters/huaweiads/huaweiads.go:774]: Hello,

Just did. Thanks!

**Review by gargcreation1992 — APPROVED** [review]
lgtm!

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**